docs/index.html

```diff
@@ -371,340 +337,685 @@ <h3>RP Meta-data</h3>
+                    interval MUST be increased by 5 seconds for this and all subsequent requests.
+                </li>
+                <li>
+                    <code>expired_token</code>: The <code>device_code</code> has expired and the device binding session
+                    has concluded. The <a href="#siop">SIOP</a> SHOULD stop polling and send the
+                    <code>&lt;SIOP DID Response&gt;</code>.
+                </li>
+                <li>
+                    <code>access_denied</code>: The end user denied the request and the <a href="#siop">SIOP</a> SHOULD
+                    stop polling and send the <code>&lt;SIOP DID Response&gt;</code>.
+                </li>
+            </ul>
+
+            <p>
+                The <code>authorization_pending</code> and <code>slow_down</code> error codes define particularly unique
+                behavior, as they indicate that the <a href="#siop">SIOP</a> should continue to poll the verification
+                URI by repeating the <code>&lt;Device Binding Request&gt;</code> (implementing the precise behavior
+                defined above).  If the <a href="#siop">SIOP</a> receives an error response with any other error code,
+                it MUST stop polling and SHOULD react accordingly, for example, by displaying an error to the user.
+            </p>
+            <p>
+                On encountering a connection timeout, <a href="#siop">SIOPs</a> MUST unilaterally reduce their polling
+                frequency before retrying.  The use of an exponential backoff algorithm to achieve this, such as by
+                doubling the polling interval on each such connection timeout, is RECOMMENDED.
+            </p>
+            <p>
+                The assumption of this specification is that the <a href="#rp">RP</a> does not have a way to communicate
+                back to the <a href="#siop">SIOP</a>. This protocol only requires a one-way channel in order to maximise
+                the viability of the protocol in restricted environments, like a <a href="#siop">SIOP</a> on a
+                mobile phone that is only capable of outbound requests. If a return channel were to exist for the chosen
+                user interaction interface, then the <a href="#siop">SIOP</a> MAY wait until notified on that channel
+                that the user has completed the action before sending the <code>&lt;SIOP DID Response&gt;</code> (as an
+                alternative to polling). Such behavior is, however, outside the scope of this specification.
+            </p>
+
+            <p>
+                For example, the <a href="#rp">RP</a> responds the following HTTPS response in case the request was
+                successful:
+            </p>
+            <pre class="example nohighlight">
+                HTTP/1.1 200 OK
+                Cache-Control: no-store
```

Comment:
Should we require (or recommend) the use of these headers, or are they purely for example purposes? (I think it's good to call out explicitly or omit from the example.)